docs/app/Examples/modules/Dropdown/Content/AsyncOptions.js

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash'
+import React, { Component } from 'react'
+import { Button, Dropdown } from 'stardust'
+import faker from 'faker'
+
+export default class DropdownAsyncOptions extends Component {
+  state = { isFetching: false }
+
+  updateItems = () => {
+    this.setState({ isFetching: true })
+
+    // fake api call
+    setTimeout(() => {
+      this.setState({
+        isFetching: false,
+        options: _.times(50, () => {
+          const name = faker.name.findName()
+          return { text: name, value: name }
+        }),
+      })
+    }, 1000)
+  }
+
+  render() {
+    const { options, isFetching } = this.state
+    return (
+      <div>
+        <Button onClick={this.updateItems}>
+          Fetch Items
+        </Button>
+        <Dropdown
+          className={`search selection multiple ${isFetching && 'disabled loading'}`}
```

Comment:
I had opened an issue related to programmatic use of the loading indicator with https://github.com/TechnologyAdvice/stardust/issues/149. Can that be closed with this PR?